aibolit/__main__.py

```diff
@@ -151,7 +151,7 @@ def __count_value(value_dict, input_params, code_lines_dict, java_file: str, is_
         )
 
 
-def flatten(l):
+def flatten(l: List[List[Any]]) -> List[Any]:
```

Comment:
Something like this:
```suggestion
def flatten(l: List[List[T]]) -> List[T]:
```

But also need to declare `T`:
``` python
from typing import TypeVar
T = TypeVar('T')
```